site/content/docs/5.1/content/reboot.md

```diff
@@ -20,7 +20,7 @@ Here are our guidelines and reasons for choosing what to override in Reboot:
 
 ## CSS variables
 
-<small class="d-inline-flex px-2 py-1 font-monospace text-muted border rounded-3">Added in v5.1.1</small>
+<small class="d-inline-flex px-2 py-1 fw-semibold text-success bg-success bg-opacity-10 rounded-2">Added in v5.2.0</small>
```

Comment:
Why has it been changed to v5.2.0? See https://getbootstrap.com/docs/5.1/content/reboot/#css-variables.